src/ui/atoms/index.ts

```diff
@@ -1,6 +1,8 @@
 import * as button from './button';
+import * as labelControl from './label-control';
```

Comment:
Давай переименуем его в `label` ?

```suggestion
import * as label from './label-control';
```